app/src/main/res/drawable/ic_edit_time_small.xml

```diff
@@ -0,0 +1,15 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:autoMirrored="true"
```

Comment:
That's not supported below API 21, but I think we don't care it doesn't get mirrored there. From a quick look it seems that we only have 0.2% of users on APIs 16-20 combined.